src/lines_lf.rs

```diff
@@ -100,16 +111,16 @@ fn to_byte_idx_impl<T: ByteChunk>(text: &[u8], line_idx: usize) -> usize {
     }
 
     // Take care of any unaligned bytes at the end.
-    let end = &text[byte_count..];
-    for byte in end.iter() {
+    for byte in &text[byte_count..] {
         if lf_count == line_idx {
             break;
         }
-        lf_count += (*byte == 0x0A) as usize;
+        if *byte == LF {
+            lf_count += 1;
+        }
```

Comment:
This also seems like just a stylistic change.  I'm not opposed to stylistic suggestions, but let's keep those separate from PRs with functional/perf changes.